libevolve/ga/SGA.py

```diff
@@ -0,0 +1,156 @@
+import random
+
+from ._base import GeneticAlgorithm
+from deap import base, algorithms, creator, tools
+from libevolve.libevolve.common._base import EvoCategoricalParam,EvoIntParam,EvoFlagParam, EvoFloatParam
```

Comment:
you should import from `libevolve.libevolve.common`
```suggestion
from libevolve.libevolve.common import *
```